src/tools/pbs_python.c

```diff
@@ -1307,39 +1320,55 @@ pbs_python_populate_svrattrl_from_file(char *input_file,
 					if (event_svrattrl)
 						rc = add_to_svrattrl_list(event_svrattrl, name_str, resc_str, val_str, 0, NULL);
 				}
-			} else if( event_job_svrattrl && \
+			} else if (event_job_svrattrl &&
 			(strcmp(obj_name, EVENT_JOB_OBJECT) == 0) ) {
-				rc = add_to_svrattrl_list(event_job_svrattrl, name_str,
-					resc_str, val_str, 0, NULL);
-			} else if( event_job_o_svrattrl && \
+				if (strcmp(name_str, PY_JOB_FAILED_MOM_LIST) == 0) {
+					if (job_failed_mom_list_svrattrl) {
+						rc = add_to_svrattrl_list(job_failed_mom_list_svrattrl, val_str, NULL, NULL, 0, NULL);
+					}
+				} else if (strcmp(name_str, PY_JOB_SUCCEEDED_MOM_LIST) == 0) {
+					if (job_succeeded_mom_list_svrattrl) {
+						rc = add_to_svrattrl_list(job_succeeded_mom_list_svrattrl, val_str, NULL, NULL, 0, NULL);
+					}
+				} else {
+					rc = add_to_svrattrl_list(event_job_svrattrl, name_str, resc_str, val_str, 0, NULL);
+				}
+			} else if( event_job_o_svrattrl &&
```

Comment:
There are more cases where you need a space after "if" and to remove backslashes. You only need backslashes for multi-line macro definitions.

Reply:
Fixed all 'if(' cases to 'if (' and the unnecessary backslashes.